docs/user/modeling.md

```diff
@@ -106,8 +116,10 @@ class NautobotIPAddress(NautobotModel):
 For "*-to-many" relationships such as (generic) foreign keys traversed backwards or [many-to-many relationships](https://docs.djangoproject.com/en/3.2/topics/db/examples/many_to_many/), we need to employ a different mechanism. Again we start by identifying which fields of the related object we are interested in for queries **to** the model. In this case, our example will be using an `Interface` model for which we also want to sync the associated IP addresses. For our scenario, lets assume that our IP addresses can be uniquely identified through the `host` and `prefix_length` fields:
 
 ```python
-from typing_extensions import TypedDict  # Python<3.9 - you need to install this module
-# from typing import TypedDict  # Python>=3.9
+try:
+    from typing import TypedDict  # Python>=3.9
+except ImportError:
+    from typing_extensions import TypedDict  # Python<3.9 - you need to install this module
```

Comment:
this should be handled in pyproject.toml:

```toml
typing_extensions = {version = "*", python = "<3.8"}
```

Reply:
Yes, but this doesn't help the person that copy pastes this example code, and they may also not be using poetry

Reply:
I had the same comment, this is just an md, not a python file so there is no update to this lib, just the example user needs to know.

Reply:
If using `typing_extensions` is necessary to take advantage of this functionality of the plugin,  then it should be (at the very minimum an optional) dependency of the plugin.